### PR TITLE
Add rtd build files and edit doc

### PR DESCRIPTION
**Pull request by willingc**

### CODE REVIEW

File: doc/environment.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: mybinder 
+channels:
+  - conda-forge
+dependencies:
+- python=3.5
+- sphinx>=1.3.6,!=1.5.4
+- pip:
+    - recommonmark==0.4.0
```
Comment (choldgraf) [inline doc/environment.yml]: why the extra spaces here?

**Comment by choldgraf** [comment]
looks great!!

File: doc/environment.yml (hunk 1 above)
Comment (willingc) [inline doc/environment.yml]: Yaml file. And recommonmark is installed by pip instead of conda.

**Comment by choldgraf** [comment]
Ah, I'm stupid haha, I didn't see the extra indent before that line

**Comment by choldgraf** [comment]
No more coding on a Friday afternoon for me :-)

**Pull request merged**